lib/converters/circuit-to-3d.ts

```diff
@@ -31,6 +30,99 @@ function convertRotationFromCadRotation(rot: {
   }
 }
 
+function getNestedValue(obj: unknown, path: string[]): unknown {
+  let current: any = obj
+  for (const key of path) {
+    if (!current || typeof current !== "object") return undefined
+    current = current[key]
+  }
+  return current
+}
+
+function extractGltfUrl(cad: CadComponent): string | undefined {
+  const candidateKeys = [
+    "model_gltf_url",
+    "model_url",
+    "gltf_url",
+    "gltfUrl",
+    "modelGltfUrl",
+    "modelUrl",
+  ] as const
+
+  for (const key of candidateKeys) {
+    const value = (cad as any)[key]
+    if (typeof value === "string" && value.length > 0) {
+      return value
+    }
+  }
+
+  const nestedPaths = [
+    ["model", "gltf_url"],
+    ["model", "gltfUrl"],
+    ["cad_model", "gltf_url"],
+    ["cad_model", "gltfUrl"],
+    ["cadModel", "gltfUrl"],
+  ]
```

Comment:
This is crazy. Gltf urls are literally ALWAYS cad_component.model_gltf_url or cad_component.model_glb_url